pkg/transfer/flex/main.go

```diff
@@ -1,13 +1,16 @@
 package main
 
 import (
+	"bytes"
 	"context"
 	"encoding/json"
-	"errors"
 	"fmt"
+	"github.com/pkg/errors"
```

Comment:
Usually, imports are organized in groups, with the packages from the standard library at the top. You can find more information [here](https://github.com/golang/go/wiki/CodeReviewComments#imports). And following your code editor, there are some extensions that order your imports for you.

Reply:
Ah, that makes more sense. I will fix my editor.